src/tests/ie_test_utils/common_test_utils/ov_tensor_utils.cpp

```diff
@@ -252,12 +252,14 @@ void compare(const ov::Tensor& expected,
             }
         }
     }
+#ifndef NDEBUG
```

Comment:
I would propose to perform this change in scope of separate PR just in case (since it is not related to the original change)

Reply:
Done